src/frontend/components/SelectContent/index.spec.tsx

```diff
@@ -0,0 +1,294 @@
+              new_video_url={appData.new_video_url}
+              lti_select_form_action_url={appData.lti_select_form_action_url!}
+              lti_select_form_data={appData.lti_select_form_data!}
+            />
+          </Grommet>,
+        ),
+      );
+
+      screen.getByTitle('Select Video 1');
+      expect(
+        screen.getByTitle('Select Video 1').getElementsByTagName('img')[0],
+      ).toHaveAttribute('src', 'https://example.com/uploaded_thumbnail/480');
+    });
+
+    it('fallback to generated video thumbnail if uploaded thumbnail not ready', async () => {
+      render(
+        wrapInIntlProvider(
+          <Grommet>
+            <SelectContent
+              videos={[
+                videoMockFactory({
+                  id: '1',
+                  title: 'Video 1',
+                  upload_state: uploadState.PROCESSING,
+                  is_ready_to_show: false,
+                  thumbnail: {
+                    active_stamp: null,
+                    is_ready_to_show: false,
+                    upload_state: uploadState.PROCESSING,
+                    id: '1',
+                    video: '1',
+                    urls: {
+                      480: 'https://example.com/uploaded_thumbnail/480',
+                      720: 'https://example.com/uploaded_thumbnail/720',
+                      1080: 'https://example.com/uploaded_thumbnail/1080',
+                    },
+                  },
+                }),
+              ]}
+              new_document_url={appData.new_document_url}
+              new_video_url={appData.new_video_url}
+              lti_select_form_action_url={appData.lti_select_form_action_url!}
+              lti_select_form_data={appData.lti_select_form_data!}
+            />
+          </Grommet>,
+        ),
+      );
+
+      screen.getByTitle('Select Video 1');
+      expect(
+        screen.getByTitle('Select Video 1').getElementsByTagName('img')[0],
+      ).toHaveAttribute('src', 'https://example.com/default_thumbnail/144');
+    });
+
+    it('select content', async () => {
+      render(
+        wrapInIntlProvider(
+          <SelectContent
+            documents={[
+              documentMockFactory({
+                id: '1',
+                title: 'Document 1',
+                upload_state: uploadState.PROCESSING,
+                is_ready_to_show: false,
+              }),
+            ]}
+            lti_select_form_action_url={appData.lti_select_form_action_url!}
+            lti_select_form_data={{
+              lti_response_url: 'https://example.com/lti',
+              lti_message_type: 'ContentItemSelection',
+            }}
+          />,
+        ),
+      );
+
+      const documentTab = screen.getByRole('tab', {
+        name: 'Documents',
+      });
+      fireEvent.click(documentTab);
+      fireEvent.click(screen.getByTitle('Select Document 1'));
+
+      expect(window.HTMLFormElement.prototype.submit).toBeCalled();
```

Comment:
I consider using `.toHaveBeenCalledTimes(number)` over `toBeCalled` as a best practice.
It avoids several issues:
1. Forgot to reset mock and have a false positive test
2. Ensure that the react render loop do not trigger this method more than expected (prevent potential performance issue)

```suggestion
      expect(window.HTMLFormElement.prototype.submit).toHaveBeenCalledTimes(1);
```